.pipelines/azure_pipeline_mergedbranches.yaml

```diff
@@ -33,11 +33,12 @@ jobs:
   ## prod release are in format Major.Minor.Patch
   ## preview release are in format preview-Major.Minor.Patch
   - bash: |
-      commit=$(git describe) 
-      linuxImagetag="$commit"
-      windowsImageTag=win-"$commit"
+      commit=$(git describe | cut -d'-' -f1,2)
```

Comment:
why are we removing the commit id? we can keep commit id and add timestamp onto that, right?

Reply:
updated, keep both id and date